frontend/src/constants/API.tsx

```diff
@@ -9,6 +9,17 @@ export interface IPaginateParams {
   sort?: string;
 }
 
+export interface IGetUsersParams extends IPaginateParams {
+  username?: string;
+  firstName?: string;
+  lastName?: string;
+  email?: string;
+  agency?: string;
+  role?: string;
+  isDisabled?: boolean | null;
```

Comment:
Why would `isDisabled` ever be null?

Reply:
I agree with @Fosol here. The boolean is already optional by way of `isDisabled?`. So this indicates the resulting object can have 3 states:

```js
isDisabled: true
isDisabled: false
isDisabled: undefined // property not present in the object
```

Why would we need a **fourth** state?

```js
isDisabled: null // present in the object with value "null"
```